q2_feature_table/_summarize/summarize_assets/sample-frequency-detail.html

```diff
@@ -1,9 +1,46 @@
-{% extends "base.html" %}
-
-{% block content %}
-  <div class="row">
-    <div class="col-lg-12">
-      {{ sample_frequencies_table }}
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+    <link rel="stylesheet" href="./q2templateassets/css/bootstrap.min.css">
+    <link rel="stylesheet" href="./q2templateassets/css/normalize.css">
+    <script src="./app/vendor.bundle.js"></script>
+    <script src="./app/js/child.js"></script>
+  </head>
+  <body>
+    <div class="row">
+      <div class="form-group col-lg-2 col-md-2 col-sm-12 col-xs-12">
+        <label for="slider-value">Sampling Depth:</label>
+        <input
+          id="slider-value"
+          class="form-control"
+          type="number"
+        >
+        <br>
+        <input
+          id="slider"
+          type="range"
+          value="0"
+          min="0"
+          max="{{ max_count }}"
+        >
+        <br>
+      </div>
+      <div id="histogram" class="col-lg-10 col-md-10 col-sm-12 col-xs-12"></div>
     </div>
-  </div>
-{% endblock %}
+    <div class="row">
+      <div id="table" class="col-lg-10 col-md-10 col-sm-12 col-xs-12 pull-right"></div>
+    </div>
+    <script type="text/javascript">
+      var m;
+      var c;
+      function load(metadata, counts) {
```

Comment:
oh is it possible that `init` isn't global because of how webpack handles imports? What if you did `window['init'] = function(){...}`